package.json

```diff
@@ -10,7 +10,7 @@
     "prepreview-build": "npm run get-submodules && ./check-hugo.sh",
     "preproduction-build": "npm run get-submodules && ./check-hugo.sh",
     "preserve": "./check-hugo.sh",
-    "preview-build": "hugo --baseURL $DEPLOY_PRIME_URL -e development --minify",
+    "preview-build": "hugo --baseURL $DEPLOY_PRIME_URL -e development --minify && make ci-check-links",
```

Comment:
@chalin, should we add this to the production build, or is the preview build enough (nothing should get built in `master` that hasn't been tested in preview...

Passing output:
<img width="1200" alt="Screen Shot 2021-04-01 at 10 12 52 AM" src="https://user-images.githubusercontent.com/4453979/113329969-156aaf00-92d3-11eb-9df3-b1e216438d22.png">

Reply:
> @chalin, should we add this to the production build, or is the preview build enough

For now, I'd merge as is with checks only for preview builds.

Reply:
Leave `preview-build` as is and add a new script:

```json
  "postpreview-build": "make ci-check-links"
```